pkg/machines/config.es6

```diff
@@ -35,7 +35,7 @@ const VMS_CONFIG = {
         }
     },
     // TODO: make it configurable via config file
-    isDev: false, // Never commit with 'true'
+    isDev: true, // Never commit with 'true'
```

Comment:
This contradicts the comment, so this should be reverted, or the comment updated with a justification.